crates/nuon/src/to.rs

```diff
@@ -110,7 +110,7 @@ fn value_to_string(
         // Propagate existing errors
         Value::Error { error, .. } => Err(*error.clone()),
         // FIXME: make filesizes use the shortest lossless representation.
-        Value::Filesize { val, .. } => Ok(format!("{}b", *val)),
+        Value::Filesize { val, .. } => Ok(format!("{}B", val.get())),
```

Comment:
This is a break in nuon output and I know @amtoine was slightly worried about those

Reply:
Oh yeah, I should save that for the other PR.

Reply:
yup that could lead to unexpected bugs when hashing NUON data for instance :wink: